.github/workflows/package-release.yml

```
@@ -54,6 +54,32 @@ jobs:
         with:
           name: dream_textures-windows-cuda
           path: dream_textures-windows-cuda.zip
+  windows-directml:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          path: dream_textures
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Install dependencies into target
+        shell: bash
+        run: 'python -m pip install -r requirements/win-dml.txt --no-cache-dir --target .python_dependencies'
+        working-directory: dream_textures
+      - name: Archive Release
+        uses: thedoctor0/zip-release@main
+        with:
+          type: zip
+          filename: dream_textures-windows-dml.zip
```

```suggestion
          filename: dream_textures-windows-directml.zip
```
The archive and upload step expects a different name.